packages/git/src/common/git.ts

```diff
@@ -900,26 +900,6 @@ export namespace GitUtils {
         return !!arg && ('paths' in arg);
     }
 
-    /**
-     * The error code for when the path to a repository doesn't exist.
-     */
-    const RepositoryDoesNotExistErrorCode = 'repository-does-not-exist-error';
-
-    /**
-     * `true` if the argument is an error indicating the absence of a local Git repository.
-     * Otherwise, `false`.
-     */
-    // tslint:disable-next-line:no-any
-    export function isRepositoryDoesNotExistError(error: any | undefined): boolean {
```

Comment:
@kittaakos is it ok to move this next to the single call site?

Reply:
thanks @kittaakos for chatting on this (offline) 💯 

I'll revert this (kind of) unrelated change to make it cleaner.

Reply:
change reverted.